app/controllers/shoping_lists_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ShopingListsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    # the list of food that is missing for all recipes of the logged-in user
+    @shoping_lists = Food.select('foods.name, sum(foods.quantity) as quantity, sum(foods.price) as price')
+      .where(user: current_user)
+      .joins(:recipe_foods, :recipes).select('count(recipes.name) as recipes')
```

Comment:
* As pointed out by the previous CR Your shopplist view is still missing with its expected features Kindly work on this 
## Your shoplist view currently 
![image](https://user-images.githubusercontent.com/52497246/188085562-65b7d074-7093-47c9-9025-d942ae529a5e.png)
## Given shoplist view wireframe 
![image](https://user-images.githubusercontent.com/52497246/188085640-98ea14e2-5158-442c-84b3-b81687c1d937.png)

Reply:
Update with 12bb8f9 commit hash.

Reply:
![Capture](https://user-images.githubusercontent.com/69077061/188095079-2e64dc9e-f913-4534-b313-52d79a277c20.PNG)